legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -129,3 +133,32 @@ def post_businesses():
             HTTPStatus.SERVICE_UNAVAILABLE
 
     return ListFilingResource.put(bootstrap.identifier, None)
+
+
+@bp.route('/affiliations', methods=['POST'])
```

Comment:
This shouldn't be called affiliations, affiliations should live in AUTH. 

Entities should live in LEAR / NAMEX